src/components/lyne-title/usage/usage.md

```diff
@@ -0,0 +1 @@
+Use it like this: `<lyne-title level="1" text="My title text"></lyne-title>`
```

Comment:
if you like, you can remove this file. that was just an example testing a functionality from my side.